I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 85, in script
    show p1s1 at middle
  File "game/script.rpy", line 85, in <module>
    show p1s1 at middle
NameError: name 'middle' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 85, in script
    show p1s1 at middle
  File "C:\Users\germe\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 1428, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\germe\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 1387, in show_imspec
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\germe\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 1387, in <listcomp>
    at_list = [ renpy.python.py_eval(i) for i in at_list ]
  File "C:\Users\germe\Downloads\renpy-8.1.3-sdk\renpy\python.py", line 1153, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\germe\Downloads\renpy-8.1.3-sdk\renpy\python.py", line 1146, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 85, in <module>
    show p1s1 at middle
NameError: name 'middle' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.3.23091805
CaveGame 1.0
Mon Nov 20 21:14:20 2023
